packages/data/src/types.ts

```diff
@@ -32,18 +38,215 @@ export interface ReduxStoreConfig<
 	initialState?: State;
 	reducer: ( state: any, action: any ) => any;
 	actions?: ActionCreators;
-	resolvers?: MapOf< Resolver >;
+	resolvers?: ResolversOf< Selectors >;
 	selectors?: Selectors;
 	controls?: MapOf< Function >;
 }
 
+/** Unsubscribes from a registered listener. */
+export interface Unsubscriber {
+	(): void;
+}
```

Comment:
Is there any reason that you're using an interface here instead of the more readable `type` declaration?

```suggestion
export type Unsubscriber = () => void;
```